src/index.ts

```diff
@@ -145,13 +145,18 @@ export function repl() {
         output: process.stdout,
     });
     rl.setPrompt("brs> ");
-
     rl.on("line", line => {
+        if (line.toLowerCase() === "quit" || line.toLowerCase() === "exit") {
+            process.exit();
+        }
         let results = run(line, defaultExecutionOptions, replInterpreter);
         if (results) {
-            results.map(result => console.log(result.toString()));
+            results.map(result => {
+                if (result.toString() !== "invalid") {
```

Comment:
Rather than comparing strings, could we compare against the `BrsInvalid.Instance` singleton?   Something like

```suggestion
                if (result !== BrsTypes.BrsInvalid.Instance) {
```

appears to work the same way and is more explicit (it also doesn't convert to a string twice 😉)